new-library.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
```

Comment:
Can we please not introduce more ruby? We already have too many requirements and this one has been causing problems #275